### PR TITLE
[PL] Partial translations in `Installation`, `Administrators`, `Rules` and `Wiki` articles

### PR DESCRIPTION
**Pull request by Zordon1337**
As title says, i've partially translated wiki to polish, unfortunately i skipped 
People->BAT
Ranking Criteria
Because i have no clue how to translate them especially the Ranking Criteria part
rest is fully translated

### CODE REVIEW

**Comment by Zordon1337** [comment]
Hello?

**Comment by Lekuruu** [comment]
Hi

**Comment by Zordon1337** [comment]
> Hi

When review :troll_face:

**Comment by Lekuruu** [comment]
When u finish it + when I learn polish

**Comment by Zordon1337** [comment]
Okay cya in 420 years 😀😀😀😀😀😀😀😀😀😀😀

pon., 26 sie 2024, 09:13 użytkownik Levi ***@***.***> napisał:

**Comment by BlueChinchompa** [comment]
polska gurom

also we should probably make a page at some point talking about translation and how we should go about approving that. (I totally wont forget about this tomorrow)

**Comment by Zordon1337** [comment]
o shit i forgot about this one

**Comment by BlueChinchompa** [comment]
o shit

**Comment by Zordon1337** [comment]
I wonder if my translation got already outdated

File: wiki/People/Administrators/pl.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Administracja
+
+Ludzie którzy zarządzają, moderują i ulepszają titanic'a każdego dnia.
+
+<!-- This should probably have more to it however sending it as is for other people to give ideas towards  -->
+
+
+## Current Members
```
Comment (iSlodinxOsu) [inline wiki/People/Administrators/pl.md:8]: You need to translate this

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/Installation/pl.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Instalacja
+
+Aby zacząć gre na Titanic'u! przejdź na stronie [pobierania clientów](https://osu.titanic.sh/download/) I wybierz dowolną wersje od 2007 roku aż do 2014
+
+<img src="img/BuildDownload.png">
+
+Po ukończeniu pobierania, jedyne co musisz zrobić to wypakować folder i otworzyć `osu!.exe`.
+
+Jest szansa na to że Windows Defender zablokuje plik z osu, ponieważ plik gry nie jest podpisany cyfrowo (Takie podpisanie by kosztowało nas 200$).
```
Comment (iSlodinxOsu) [inline wiki/Installation/pl.md:9]:
```suggestion
Jest szansa na to że Windows Defender zablokuje plik z osu!, ponieważ plik gry nie jest podpisany cyfrowo (Takie podpisanie by kosztowało nas 200$).
```

File: wiki/People/Administrators/pl.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+Użytkownik | Rola
+---|:---:
+[Levi](https://osu.titanic.sh/u/2)             | Główny Deweloper i Menedżer projektu
+[Adachi](https://osu.titanic.sh/u/39)          | Menadżer Społecznościowy i Deweloper systemu PP
```
Comment (iSlodinxOsu) [inline wiki/People/Administrators/pl.md:13]:
```suggestion
[Adachi](https://osu.titanic.sh/u/39)          | Menadżer Społecznościowy i Deweloper systemu pp
```

File: wiki/Rules/pl.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Zasady
+
+Witaj na titanic'u, zanim zaczniesz proszę przeczytaj te zasady, by cieszyć się bezpieczniejszym i przyjemniejszym doświadczeniem.
+
+# Zasady Społeczności
+
+1. **Każdy użytkownik może posiadać *ONE* konto KIEDYKOLWIEK!** Jedynym wyjątkiem jest robienie konto do botów lub innych projektow, ale najpierw musisz poinformować administracja żebyśmy mogli to zatwierdzić. Jeżeli nie masz dostępu do maila albo zapomniałeś hasła, proszę otwórz ticket na [Serwerze discord](https://discord.gg/qryYG2C5nc).
+2. **Graj uczciwie.** Używanie jakichkolwiek narzędzi trzecich które dają nieuczciwą przewage nad innymi graczami skończy sie blokadą konta. Obejmuje to liste programów typu makro, cheat engine, wspomagacze celowania, timewarp, itd. Jeżeli widzisz jakąś podejrzaną aktywność ze strony użytkownika, proszę otwórz ticket na [Serwerze discord](https://discord.gg/qryYG2C5nc).
+3. **Bądz miły dla innych.** Jakiekolwiek zastraszanie, raszim, lub antagonizm nie ma miejsca tutaj na Titanic'u! To jest poprostu serwer o klikaniu kółek, łapaniu owoców, uderzania bębny, i rozwalaniu klawiatury podczas udawania że jesteś cookiezi'm i graniu starych map, nie żeby być dupkiem wobec innych.
```
Comment (iSlodinxOsu) [inline wiki/Rules/pl.md:9]:
```suggestion
3. **Bądz miły dla innych.** Jakiekolwiek zastraszanie, raszim, lub antagonizm nie ma miejsca tutaj na Titanic'u! To jest poprostu serwer o klikaniu kółek, łapaniu owoców, uderzania bębny, i rozwalaniu klawiatury podczas udawania że jesteś Cookiezi'm i graniu starych map, nie żeby być dupkiem wobec innych.
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

**Comment by Zordon1337** [comment]
Maybe today

**Comment by iSlodinxOsu** [comment]
By the way, just to remind you to update your PR, due to these changes in https://github.com/osuTitanic/wiki/pull/33/

**Comment by Lekuruu** [comment]
I'd close this for now. @Zordon1337 if you get time again for a translation, please make a pr again, since a lot of pages have been updated.

**Pull request closed without merging**